roles/keystone/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 keystone:
-  rev: 8a7282bebd1ebfb92d9ccd87f85b8490a89da4e0
+  rev: 5db37331e6ea3b21314d4d47e63fa1505e48285a
```

Comment:
a) pip < 1.5 has a bug that breaks installing from a pip mirror behind a firewall
b) pip > 1.4 can't be used to install the revs we were using because of busted sqlalchemy version restrictions.

therefore I had to bump the versions to have sane sqlalchemy reqs so I can use the newer pip so I can install behind the firewall.  I bumped them to the latest juno stable commit.